app/src/interfaces/list-o2m/list-o2m.vue

```diff
@@ -49,6 +49,7 @@
 				:loading="loading"
 				:items="displayItems"
 				:row-height="tableRowHeight"
+				:server-sort="pageCount > 1"
```

Comment:
Should we just rely on server-sort always? Even though it'll be a hair slower, it makes the overall codebase quite a bit cleaner 🙂

Reply:
That is possible, i was aiming for some performance edge cases as the server sort fires 2 requests on each sort action ( a count and an item fetch) which may take a while if the table these records come from is huge. Seemed like a waste to make a roundtrip to the server while we already have the logic and data to sort on the frontend too if there is only 1 page.

Reply:
Why does it fetch count on a sort? Can't we skip that if we already know the count of items?

Reply:
Very good question it shouldnt have to 🤔 This happens in the abstracted `useRelationMultiple` function which is build to also work with filters for which it would need to fetch the count even tho not used in this situation. I'll have a look to see if i can make that logic smarter.